doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -125,23 +125,23 @@ Traceability to Reference Model.
 
 | Ref # | sub-category | Description |  Traceability |
 |----|----------|--------------------|------------|
-| `req.int.api.01` | API | The Architecture **must** provide Control API endpoints to cloud platform core services. | |
-| `req.int.api.02` | API | The Architecture **must** provide GUI access to tenant facing cloud platform core services. | |
-| `req.int.api.03` | API | The Architecture **must** provide APIs needed to discover and manage NFVI resources. | |
-| `req.int.acc.01` | Acceleration | The Architecture **should** provide an open and standard acceleration interface to VNFs. | |
+| `req.int.api.01` | API | The Architecture **must** provide Control API endpoints to cloud platform core services. | [RA-1 5.3. "Consolidated Set of APIs"](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter05.md#53-consolidated-set-of-apis) |
+| `req.int.api.02` | API | The Architecture **must** provide GUI access to tenant facing cloud platform core services. | [RA-1 4.3.1.9 "Horizon"](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter04.md#4319-horizon) |
+| `req.int.api.03` | API | The Architecture **must** provide APIs needed to discover and manage NFVI resources. | [RA-1 5.2.7. "Placement"](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter05.md#527-placement) |
+| `req.int.acc.01` | Acceleration | The Architecture **should** provide an open and standard acceleration interface to VNFs. | [RA-1 5.3.4. "Cyborg"](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter05.md#534-cyborg) |
 | `req.int.acc.02` | Acceleration | The Architecture **should not** rely on SR-IOV PCI-Pass through for acceleration interface exposed to VNFs. | |
 
 <p align="center"><b>Table 2-4:</b> Interfaces and APIs Requirements</p>
 
 <a name="2.3.5"></a>
-### 2.3.5 Tenants Requirements
+### 2.3.5 Tenant Requirements
 
 | Ref # | sub-category | Description |  Traceability |
 |----|--------------|---------------|-----------------|
-| `req.tnt.gen.01` | General | The Architecture **must** support multi-tenancy. | |
-| `req.tnt.gen.02` | General | The Architecture **must** support self-service dashboard (GUI) and APIs for users to deploy, configure and manage their workloads. | |
+| `req.tnt.gen.01` | General | The Architecture **must** support multi-tenancy. |  **duplicate of req.vim.07** |
+| `req.tnt.gen.02` | General | The Architecture **must** support self-service dashboard (GUI) and APIs for users to deploy, configure and manage their workloads. | [RA-1 4.3.1.9 "Horizon"](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter04.md#4319-horizon) and [3.3.1.4 Cloud Workload Services](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter03.md#3314-cloud-workload-services) |
```

Comment:
Is `req.tnt.gen.02` is also covered by `req.int.api.02` and `req.int.api.03`?

Reply:
@deebhatia req.tnt.gen.02 is more about users being able to utilize what the other two requirements (req.int.api.01 and req.int.api.02) make available. Do you agree?

Reply:
Okay. But does 'tenant requirement' is a correct section for this?